app/server/tests/integration/errors.test.ts

```diff
@@ -43,19 +44,18 @@ describe("Error handling", () => {
         testSample.projectId = projectId;
         const poppunkRes = await post(`poppunk`, testSample, connectionCookie);
         expect(poppunkRes.status).toBe(200);
-        const counter = 0;
+        let counter = 0;
         let finished = false;
         while (!finished && counter < 100) {
-            await new Promise(resolve => {
-                setTimeout(() => {resolve(""), 1000})
-            });
+            await setTimeout(2000);
```

Comment:
The previous attempts at delays between polling hadn't been working, so we were getting some intermittent test failures, which I think were related to spamming the endpoint. Bumping up the delay as well to avoid intermittent errors on first poll .

Reply:
can we use fake timers here instead if possible?